tasks/dist.js

```diff
@@ -39,16 +40,21 @@ module.exports = function(gulp, config) {
 
 	});
 
-	var buildTasks = ['build:dist:scripts'];
+	var buildTasks = ['build:dist:scripts', 'build:dist:css'];
 
-	if (config.component.less) {
-		gulp.task('build:dist:css', ['clean:dist'], function() {
-			return gulp.src(config.component.less)
-				.pipe(less())
-				.pipe(gulp.dest('dist'));
-		});
-		buildTasks.push('build:dist:css');
-	}
+
+	gulp.task('build:dist:css', ['clean:dist'], function() {
+
+		return gulp.src('./' + config.component.src + '/' + config.component.name + '.less')
+			.pipe(gulp.dest('dist'))
```

Comment:
Wanted to keep the less file, so that people can import it directly into their less stylesheets instead of adding the css file separately.